### ISSUE TITLE
[Feature Request] "Column View" for the Files section

### ISSUE THREAD

**Issue by Tazintosh** [mierau/hotline#20]
Hi,

I guess everything is on the tile.
Like in the Finder, would it be possible to browse the files in column view rather than a list, as a UI option?

Thank in advance for the consideration.
Cheers!

**Comment by mierau** [comment]
Yeah it probably would be. It's a bit of an undertaking but I'll think on it thanks!
